<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VuoEditorAboutBox</class>
 <widget class="QDialog" name="VuoEditorAboutBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Vuo</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QLabel" name="iconLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="VuoEditorApp/VuoEditorApp.qrc">:/Icons/vuo.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>-20</y>
      <width>521</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans</family>
     </font>
    </property>
    <property name="text">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>265</y>
      <width>221</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans</family>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <column>
     <property name="text">
      <string>Read More</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="textEdit">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>265</y>
      <width>531</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans</family>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="html">
     <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PT Sans'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Lucida Grande';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="openLinks">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="VuoEditorApp/VuoEditorApp.qrc"/>
 </resources>
 <connections/>
</ui>
